quiz-app/src/components/api/index.js

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios';
+
+// export const getCatData1 = (dispatch) => {
+//     return (dispatch)=> {
+//         axios.get('https://opentdb.com/api_category.php')
+//         .then((response) => {
+//             const categoryData = response.data;
+//             dispatch({ type: 'catDropDown', payload: categoryData});
+//         })
+//     }
+    
+// }
+
+export const getCatData = (dispatch) => {
```

Comment:
In this file, could you comment on what this block of code is doing?

Reply:
Noted.